src/osx/cocoa/utils.mm

```diff
@@ -73,8 +73,16 @@ - (void)applicationDidFinishLaunching:(NSNotification *)notification
     [NSApp stop:nil];
     wxTheApp->OSXOnDidFinishLaunching();
 
-    [NSApp setActivationPolicy:NSApplicationActivationPolicyRegular];
-    [NSApp activateIgnoringOtherApps: YES];
+    CFURLRef url = CFBundleCopyBundleURL(CFBundleGetMainBundle() ) ;
+    CFStringRef path = CFURLCopyFileSystemPath ( url , kCFURLPOSIXPathStyle ) ;
+    CFRelease( url ) ;
+    wxString app = wxCFStringRef(path).AsString(wxLocale::GetSystemEncoding());
+    // Is only needed for non-bundled apps
+    if ( !app.EndsWith(".app") )
+    {
+        [NSApp setActivationPolicy:NSApplicationActivationPolicyRegular];
+        [NSApp activateIgnoringOtherApps: YES];
```

Comment:
very minor: in this new line of code there's a space after ":" (I of course had the same from copy/pasting the original).